### ISSUE TITLE
Need help installing

### ISSUE THREAD

**Issue by ssu-tone** [Rikorose/DeepFilterNet#104]
really need step-by-step instruction, please help me 
I'm using Win10, Anaconda prompt to run the code

I followed README.md to download the code

pip install torch torchaudio -f https://download.pytorch.org/whl/cpu/torch_stable.html

and

pip install deepfilternet

works smoothy, but having issue on

**deepFilter path/to/noisy_audio.wav** 

here's the error:
2022-06-15 14:49:09 | INFO     | DF | Running on torch 1.11.0+cpu
2022-06-15 14:49:09 | INFO     | DF | Running on host DESKTOP-RP8O01C
fatal: not a git repository (or any of the parent directories): .git
2022-06-15 14:49:09 | INFO     | DF | Loading model settings of DeepFilterNet2
2022-06-15 14:49:09 | INFO     | DF | Using DeepFilterNet2 model at anaconda3\lib\site-packages\pretrained_models\DeepFilterNet2
2022-06-15 14:49:09 | INFO     | DF | Initializing model `deepfilternet2`
2022-06-15 14:49:10 | INFO     | DF | Found checkpoint anaconda3\lib\site-packages\pretrained_models\DeepFilterNet2\checkpoints\model_96.ckpt.best with epoch 96
2022-06-15 14:49:10 | **WARNING**  | DF | **Unexpected key: erb_comp.c**
2022-06-15 14:49:10 | **WARNING**  | DF | **Unexpected key: erb_comp.mn**
2022-06-15 14:49:10 | INFO     | DF | Model loaded
Traceback (most recent call last):
  File "C:\Users\Mistorm\anaconda3\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Mistorm\anaconda3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Mistorm\anaconda3\Scripts\deepFilter.exe\__main__.py", line 7, in <module>
  File "C:\Users\Mistorm\anaconda3\lib\site-packages\df\enhance.py", line 329, in run
    main(parser.parse_args())
  File "C:\Users\Mistorm\anaconda3\lib\site-packages\df\enhance.py", line 41, in main
    audio, meta = load_audio(file, df_sr)
  File "C:\Users\Mistorm\anaconda3\lib\site-packages\df\enhance.py", line 188, in load_audio
    info: AudioMetaData = ta.info(file, **ikwargs)
  File "C:\Users\Mistorm\anaconda3\lib\site-packages\torchaudio\backend\soundfile_backend.py", line 103, in info
    sinfo = soundfile.info(filepath)
  File "C:\Users\Mistorm\anaconda3\lib\site-packages\soundfile.py", line 438, in info
    return _SoundFileInfo(file, verbose)
  File "C:\Users\Mistorm\anaconda3\lib\site-packages\soundfile.py", line 383, in __init__
    with SoundFile(file) as f:
  File "C:\Users\Mistorm\anaconda3\lib\site-packages\soundfile.py", line 629, in __init__
    self._file = self._open(file, mode_int, closefd)
  File "C:\Users\Mistorm\anaconda3\lib\site-packages\soundfile.py", line 1183, in _open
    _error_check(_snd.sf_error(file_ptr),
  File "C:\Users\Mistorm\anaconda3\lib\site-packages\soundfile.py", line 1357, in _error_check
    raise RuntimeError(prefix + _ffi.string(err_str).decode('utf-8', 'replace'))
RuntimeError: **Error opening 'path/to/noisy_audio.wav': System error.**

I don't know why it can't open path/to/noisy_audio.wav
so I can't do the further step either:

(base) C:\Users\Mistorm>cd path/to/DeepFilterNet/
The system cannot find the path specified.

(base) C:\Users\Mistorm>cd DeepFilterNet
The system cannot find the path specified.

I can't figure why it can't find the file, since DeepFilterNet  and DeepFilterNet2 has already in 
C:\Users\Mistorm\anaconda3\Lib\site-packages\pretrained_models

what step did I miss? 
and what to do after that? I have a hard time understanding how to make it works...

**Comment by Rikorose** [comment]
Do you really have an audio file at `path/to/noisy_audio.wav`?

There is a step by step guide, but you need to read what you are copy pasting.

**Comment by ssu-tone** [comment]
there's no path/to/noisy_audio.wav in my files
I've searched C:\Users\Mistorm\anaconda3\Lib\site-packages, can't find it anywhere

I thought it suppose to be downloaded after the first two steps
pip install torch torchaudio -f https://download.pytorch.org/whl/cpu/torch_stable.html
pip install deepfilternet

did I miss something? where can I download it?

> Do you really have an audio file at `path/to/noisy_audio.wav`?
> 
> There is a step by step guide, but you need to read what you are copy pasting.

there's no path/to/noisy_audio.wav in my files
I've searched C:\Users\Mistorm\anaconda3\Lib\site-packages, can't find it anywhere

I thought it suppose to be downloaded after the first two steps
pip install torch torchaudio -f https://download.pytorch.org/whl/cpu/torch_stable.html
pip install deepfilternet

did I miss something? where can I download it?
